### DISCUSSION TITLE
There's nothing in Integrations.

### DISCUSSION THREAD

**Discussion by silverfantacy** [heyform/heyform#30] [Self hosting]
I'm using zeabur deployment, but there is nothing in Integrations, is there something not configured?
<img width="1067" alt="截圖 2024-04-06 晚上7 19 40" src="https://github.com/heyform/heyform/assets/20947392/08fae24e-ac20-4deb-b9d5-06038d9c6551">

**Comment by iMuFeng** [comment]
Integrations are missing prior to v0.0.5. HeyForm docker image version of >=0.0.5 is required.

**Reply by silverfantacy** [reply]
it's been updated 0.0.5 yet?
<img width="391" alt="image" src="https://github.com/heyform/heyform/assets/20947392/aa335fd6-ff7f-4e29-9e22-5a896d51cb84">

**Reply by iMuFeng** [reply]
If the database migration is not working after restarting the app, you can resolve this issue by going to the HeyForm docker container and running the migration command "cd /app && npm run migrate:seed". You will find four integrations available there.

<img width="1624" alt="heyform integrations" src="https://github.com/heyform/heyform/assets/1407009/354985c8-103f-4e60-9708-88816360c9f1">

Here is the logs:

![image](https://github.com/heyform/heyform/assets/1407009/083c5cd9-a9ac-491b-861d-c9c20b40f719)
